content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -263,9 +232,42 @@ You can now start your application locally and test the creation of log entries.
    }
    ```
 
-3. Reload the page a few times.<br /><br />Every time you reload, the script generates a new log entry and the id of the current log entry increments. If you open the [Data Browser](/data-platform/classic-projects/platform/data-browser) for your project in Prisma Data Platform, you can verify that the application writes `Info` level logs to your database.
+3. Reload the page a few times.<br /><br />Every time you reload, the script generates a new log entry and the id of the current log entry increments.
+
+   To check whether Accelerate is working, add the `withAccelerateInfo` at the end of your query. This returns an [`info`](/data-platform/accelerate/api-reference#return-type) object which provides additional information on the result of your query.
```

Comment:
Isn't the query working enough for that?

Reply:
@janpio It is. This section can be improved further.

Reply:
Since the extension is not necessary, I've removed this section